target/linux/brcm47xx/image/Makefile

```diff
@@ -403,7 +434,8 @@ endef
 TARGET_DEVICES += linksys-wrt54g
 
 define Device/linksys-wrt54gs
-  DEVICE_TITLE := Linksys WRT54GS
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := WRT54GS
   DEVICE_PACKAGES := kmod-b43
```

Comment:
If below is v4, this should be v1 (?) ...

Reply:
Let's leave it to get this ready for merge and make it explicit when know more.

Reply:
https://en.wikipedia.org/wiki/Linksys_WRT54G_series#WRT54GS suggests that the serial-number prefix can be used to differentiate the models

Reply:
I see no wrt54gs v4 in the Table Of Hardware: https://openwrt.org/toh/views/toh_fwdownload
This is probably more complicated and should fixed at a later time.